go/pkg/command/command.go

```diff
@@ -0,0 +1,273 @@
+// Package command defines common types to be used with remote execution.
+package command
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"sort"
+	"time"
+
+	"github.com/bazelbuild/remote-apis-sdks/go/digest"
+	"github.com/pborman/uuid"
+)
+
+// InputKind can be specified to narrow down the matching for a given input path.
+type InputKind int
+
+const (
+	// Any input type will match.
+	UnspecifiedInputKind InputKind = iota
+
+	// Only directories match.
+	DirectoryInputKind
+
+	// Only files match.
+	FileInputKind
+)
+
+func (s InputKind) String() string {
+	return [...]string{"UnspecifiedInputKind", "DirectoryInputKind", "FileInputKind"}[s]
+}
+
+// InputExclusion represents inputs to be excluded from being uploaded for remote execution.
```

Comment:
Same here. Not necessary uploaded, but mounted into docker for local execution for instance.

Reply:
Right.